src/main/webapp/WEB-INF/tags/admin/home/adminCreateInstructorAccountForm.tag

```diff
@@ -7,6 +7,9 @@
 
 <div class="well well-plain">
     <div>
+        <label class="label-control">Adding a Single Instructor</label>
+    </div><br/>
```

Comment:
Leaving it inline may not break the rule but I think putting it to a new line looks better. Let's do the same for below.